test/images/stub-device-plugin/pod.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: stub-device-plugin
+  labels:
+    app: stub-device-plugin
+spec:
+  volumes:
+  - name: device-plugin-path
+    hostPath:
+      path: /var/lib/kubelet/device-plugins
+  - name: dummy-device-path
+    hostPath:
+      path: /tmp/dummy-device
+  containers:
+  - name: stub-device-plugin
+    image: cargo.caicloud.io/cenph/stub-device-plugin-amd64:1.0
```

Comment:
if we make the decision to keep this yml's then we need to change this image to `gcr.io` location.